src/agent/README.md

```diff
@@ -113,3 +113,43 @@ agent control tool, rather than the Kata runtime.
 
 For further details, see the
 [agent control tool documentation](../tools/agent-ctl/README.md#run-the-tool-and-the-agent-in-the-same-environment).
+
+## Agent options
+
+The kata agent has the ability to configure agent options in guest kernel command line at runtime. Currently, the following agent options can be configured: 
+
+| Option | Description |  Value | Type | Default |
```

Comment:
I still find this slightly awkward to read. I'd be tempted to switch the order a little. I'd also consider adding a Notes column as that makes it quicker and easier to read the description and if you want further details, you can look across at the notes column. For example:

| Option | Name | Description | Type | Default | Notes |
|-|-|-|-|-|-|
| `agent.debug_console` | Debug console flag | Allow connection to guest OS running inside hypervisor | boolean | `false` | Connect using `kata-runtime exec <sandbox-id>` |

Reply:
Thanks for your comments! I have made changes.